cache-config/t3c-apply/torequest/torequest.go

```diff
@@ -1030,6 +1034,45 @@ func (r *TrafficOpsReq) ProcessPackages() error {
 	return nil
 }
 
+func pkgMetaDataToMap(pmd []string) map[string]bool {
+	pkgMap := map[string]bool{}
+	for _, pkg := range pmd {
+		pkgMap[pkg] = true
+	}
+	return pkgMap
+}
+
+func pkgMatch(pkgMetaData []string, pk string) bool {
+	for _, pkg := range pkgMetaData {
+		if ok := strings.Contains(pk, pkg); ok {
```

Comment:
you don't need to change this, but note that the assignment and checking of the variable `ok` is useless; the line is absolutely equivalent to just `if strings.Contains(pk, pkg) {`

Reply:
changed. `strings.Contains()` returns a bool. Asleep at the wheel I suppose.😀